covalent_slurm_plugin/slurm.py

```diff
@@ -118,7 +122,9 @@ def __init__(
         self.address = address or get_config("executors.slurm.address")
 
         self.ssh_key_file = ssh_key_file or get_config("executors.slurm.ssh_key_file")
-        self.ssh_key_file = str(Path(ssh_key_file).expanduser().resolve())
+        self.ssh_key_file = (
```

Comment:
avoid any calls to `expanduser` and `resolve` in the constructor since they may result in incorrect values for a remote covalent server

Reply:
Makes sense. There were two other instances where `expanduser` and `resolve` were in the constructor. I removed those too now.